packages/client/src/i18n/nl.ts

```diff
@@ -98,7 +98,7 @@ export default {
         "enter address": "Invullen adres",
       },
       map: {
-        heading: "Locatiegegevens",
+        heading: "Waar staat de boom?",
```

Comment:
Might as well put this inline in the component. The idea of translations is to reuse them. I think namespacing them like this can be improved. I'll add it to our dev-meeting topics.

Reply:
Let's discuss, i'm curious to your input here

Reply:
We'll discuss next dev meeting, for now i'll mark this as resolved on this issue